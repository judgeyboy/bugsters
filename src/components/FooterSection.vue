<template>
  <div class="footer-section">
    Made with <span class="text-error">♥</span> by <span class="footer-section_bugsters">Bugsters</span> <br>
    © 2018
  </div>
</template>

<script>
export default {
  name: 'FooterSection'
}
</script>

<style lang="sass" scoped>
.footer-section
  text-align: center
  padding: 3rem 0
  color: #acb3c2

  &_bugsters
    color: #667189
</style>
