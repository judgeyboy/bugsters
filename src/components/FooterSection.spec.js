import { shallow } from 'vue-test-utils'

import FooterSection from './FooterSection'

describe('FooterSection Component', () => {

  it('should be a Vue instance', () => {
    const wrapper = shallow(FooterSection)
    expect(wrapper.isVueInstance()).toBe(true)
  })

  it('should match snapshot', () => {
    const wrapper = shallow(FooterSection)
    const $html = wrapper.vm.$el.outerHTML
    expect($html).toMatchSnapshot()
  })

})
