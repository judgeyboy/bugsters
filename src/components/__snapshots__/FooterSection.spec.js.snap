// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterSection Component should match snapshot 1`] = `
"<div class=\\"footer-section\\">
  Made with <span class=\\"text-error\\">♥</span> by <span class=\\"footer-section_bugsters\\">Bugsters</span> <br>
  © 2018
</div>"
`;
